mpp/build.gradle.kts

```diff
@@ -130,9 +130,9 @@ tasks.register("testWeb") {
     dependsOn(":compose:runtime:runtime:jsTest")
 }
 
-tasks.register("testAll") {
-    dependsOn(":mpp:testComposeJbDesktop")
-    dependsOn(":mpp:testComposeJbWeb")
+tasks.register("testAll") { //todo maybe deprecated?
```

Comment:
Yeah, noticed that this is not really needed, let's remove it

Reply:
Removed